tests/spec/core/linter-rules/check-charset-spec.js

```diff
@@ -1,16 +1,12 @@
+import { rule } from  '../../../../src/core/linter-rules/check-charset.js';
```

Comment:
I still wonder if we can import these with “/src/core/...”? If not, we need to figure out why.

Reply:
The `commonjs` module is needed for such imports , jest doesn't work with `amd` hence the first change...

Reply:
> I still wonder if we can import these with “/src/core/...”? If not, we need to figure out why.

We need the path to the file, otherwise it just returns an error saying it cannot find the file.

Reply:
Could you try this for me:
```
console.log('../../../../src/core/linter-rules/check-charset.js', document.location); 
```

Interested to see what the whole URL is. From that, we should be able to determine the full path.

Reply:
Fixed the above... forgot the URL bit... my brain is not working today.